docs/src/tutorials/tutorial-2.md

```diff
@@ -48,16 +48,15 @@ Let's focus on the `increment` instruction, specifically the `Increment` struct
 pub struct Increment<'info> {
     #[account(mut, has_one = authority)]
     pub counter: Account<'info, Counter>,
-    #[account(signer)]
-    pub authority: AccountInfo<'info>,
+    pub authority: Signer<'info>,
 }
 ```
 
 Here, several `#[account(..)]` attributes are used.
 
-* `mut`: tells the program to persist all changes to the account.
-* `has_one`: enforces the constraint that `Increment.counter.authority == Increment.authority.key`.
-* `signer`: enforces the constraint that the `authority` account **signed** the transaction.
+- `mut`: tells the program to persist all changes to the account.
```

Comment:
Is this a functional change or is it just style?

Reply:
Thanks for the quick comment. I am a bit confused why my ide made any changes to this file besides 50-52.. Let me know if I should change them back

Reply:
No problem I was just curious if there was a functional difference. I dont think there is.